src/NUnitFramework/nunitlite.runner/AutoRun.cs

```diff
@@ -102,9 +102,9 @@ public int Execute(string[] args)
             }
 #endif
 
-            _textUI.DisplayTestFiles(new string[] { callingAssembly.GetName().Name });
             _textUI.DisplayRuntimeEnvironment();
-            _textUI.DisplayRequestedOptions();
+            _textUI.DisplayTestFiles(new string[] { callingAssembly.GetName().Name });
+//            _textUI.DisplayRequestedOptions();
```

Comment:
Why was this line commented out here, but not in Portable? If we are not going to display it, should we remove the method, or are you waiting for the where clause info to be added?